tests/cases/conformance/jsdoc/declarations/jsDeclarationsExportDoubleAssignmentInClosure.ts

```diff
@@ -8,5 +8,8 @@ function foo() {
     module.exports = exports = function (o) {
         return (o == null) ? create(base) : defineProperties(Object(o), descriptors);
     };
-    exports.methods = methods;
-}
\ No newline at end of file
+    const m = function () {
+        // I have no idea what to put here

```

Comment:
I added `m` here because otherwise the generated code was `export { unknown as methods }`, which is correct, and better than before, but caused an error in the emitted .d.ts that I didn't want.